docs/docs/firmware/tp32.md

```diff
@@ -1085,12 +1085,15 @@ Explanation text goes here...
     {
       "screen": <number>,
       "tile": <number>,
-      "image": "<encodeBase64(.png)>",
-      "zoom": <number>,
-      "posOffset":[
-        <x number>,
-        <y number>
-      ]
+      "bgImage": {
+        "base64": "<encodeBase64(.png)>",
+        "zoom": <number>,
+        "angle": <number>,
+        "posOffset":[

```

Comment:
Is this now `offset`?

Reply:
Sorry I can't follow? Do you mean the sequence of keys?

Reply:
I thought you renamed `posOffset` -> `offset` in that latest change?

Reply:
can't recall ..., The code says `posOffset `

Reply:
This is from the description in your PR...

```json
{
  "tiles":[
    {
      "screen": <number>,
      "tile": <number>,
      "bgImage": 
      {
        "base64": "<encodeBase64(.png)>",
        "zoom": <number>,
        "angle": <number>
        "offset": [<x number>,<y number>],
      }
    }
  ]
}
```

Reply:
Ah but the code doesn't match...

```cpp
tile->alignBgImage(json["bgImage"]["zoom"], json["bgImage"]["posOffset"][0], json["bgImage"]["posOffset"][1], json["bgImage"]["angle"]);
```

Reply:
I prefer `offset` but up to you whether you want to change

Reply:
I surely overlook that ..., I'll add it to the next release

Reply:
PR pushed to update to `offset` :)